### PR TITLE
Changes related to demo playback

### PR DESCRIPTION
**Pull request by meag**
Mostly changes to let me watch/re-record old demos

NQ demos: player colours set correctly, particle trails/effects back in, demo_jump working, can force team skins with help of cl_demoteamplay
QW demos: can now playback from 2.1 (protocol versions 24/25), demo rewinding works
MV demos: cam_pos stops tracking, if in free camera mode, camera position is restored after rewinding
Video capture: Can split .avi files to be stitched together later (workaround for http://sourceforge.net/p/ezquake/bugs/520/), demo_capture_quiet option to disable playback during capture.
Demo browser: plays the file you select when you have two files with valid demo extensions in same directory

### CODE REVIEW

File: cl_nqdemo.c [hunk 1]
```diff
@@ -291,10 +325,37 @@ static void NQD_ParseUpdatecolors (void)
 	// fill in userinfo values
 	top = min(colors & 15, 13);
 	bottom = min((colors >> 4) & 15, 13);
+
+	if (cl_demoteamplay.integer)
+		cl.players[i].spectator = (top == 0 && bottom == 0 && cl.players[i].frags <= -99);
```
Comment (JoakimSoderberg) [inline cl_nqdemo.c]: Would be better to have this as an `is_spectator` function... You use the same code earlier as well

**Comment by JoakimSoderberg** [comment]
Looks good to me. How about you @jite ?

**Comment by jite** [comment]
Yeah I'm gonna test it today and hopefully merge.

**Comment by jite** [comment]
Okey I've tested it quickly and it plays back everything fine, however something else has broken. Timedemo doesn't work as intended. timedemo demo1 reports +30000 frames and plays much slower than it should, so does any other demo (should be about 960 frames).

You can also test by creating your own QWD with qwdrecord and then run timedemo on that with this code and without, big difference.

I also noticed a lot more stuttering when playing back qwd's normally with this code than without.

@meag Can you take a look at this and see if you can find the culprit? Thanks

**Comment by meag** [comment]
The culprit was in cl_main/MinPhysFrameTime()... check for mvdplayback that shouldn't have been there.
Have moved spectator-checking logic into a common function, and fixed bug where cls.demotime could be less than demostarttime in NQ demos, breaking democlock.

**Pull request merged**